app/assets/stylesheets/components/_mfa-added-badge.scss

```diff
@@ -0,0 +1,20 @@
+@use 'uswds-core' as *;
+
+.usa-checkbox__input--tile + .usa-checkbox__label {
```

Comment:
Can we use the existing tag components?

https://federalist-340d8801-aa16-4df5-ad22-b1e3e731098e.sites.pages.cloud.gov/site/18f/identity-design-system/tags/

Maybe combined with a [background color utility class](https://designsystem.digital.gov/utilities/color/#background-color) if we need a different color?

Reply:
Yes. I have removed that scss and put the styles into utility classes respective of the tag component.